storage/src/vespa/storage/storageserver/communicationmanager.cpp

```diff
@@ -492,8 +489,8 @@ void
 CommunicationManager::enqueue(std::shared_ptr<api::StorageMessage> msg)
 {
     assert(msg);
-    LOG(spam, "Enq storage message %s, priority %d", msg->toString().c_str(), msg->getPriority());
-    _eventQueue.enqueue(std::move(msg));
+    LOG(spam, "Process storage message %s, priority %d", msg->toString().c_str(), msg->getPriority());
+    process(msg);
```

Comment:
We should probably rename `CommunicationManager::enqueue` in due time as well 🙂

Reply:
Yes and some other  stuff could go away too.